src/posts/server-running-p.rst

```diff
@@ -0,0 +1,69 @@
+((title . "Determining if the server is started, or the wonders of server-running-p")
+ (date . "2018-04-01 13:43:26 +02:00"))
+
```

Comment:
Please add some kind of note or disclaimer that this is a guest post and point out your nick, full name, whatever, ideally with a link to something identifying you. Otherwise this might be mistaken for me.